### ISSUE TITLE
Too big font and interface sizes 7.7.0.

### ISSUE THREAD

**Issue by uvhat** [Bionus/imgbrd-grabber#2552]
Can't get how the interface scaling works. It always too large and don't take scaling that was set

here is comparison nightly and 7.7.0 

<img width="1498" alt="interface scaling 770" src="https://user-images.githubusercontent.com/31294363/147711755-1f6de2ac-7721-4c1f-ae37-ee3c8e1239b7.png">
.

**Comment by Bionus** [comment]
What Windows zoom settings are you using?
You can disable "font scaling" in that settings window

**Comment by uvhat** [comment]
150%

As you can see, all is ok in nightly or older builds, issue persists in 7.7.0
 
"Scale font size" option do nothing, fonts still are very big.

<img width="1041" alt="interface scaling 770 disabled" src="https://user-images.githubusercontent.com/31294363/147799127-2e4ab973-ba39-4c48-965f-8fd4cfc4b26a.png">
<img width="1258" alt="interface scaling 770 enabled" src="https://user-images.githubusercontent.com/31294363/147799128-7604f72e-09ce-41da-9de4-e6de7e6f6a2d.png">

**Comment by Bionus** [comment]
That seems like intended behavior? It's more like the nightly is too small.
The old versions did not respect Windows zoom (and high-DPI scaling in general), now it does, so it's 50% bigger.
Aren't other applications doing the same? If you compare with the Windows default apps size, is there a major difference? (especially for font size)

The only difference being the "scale font size". In your screenshots, there's a slight difference (Windows default, likely 8.75pt vs 9pt).

**Comment by uvhat** [comment]
all previous is fine.
7.7.0 is bigger then any older, and bigger then another apps styles and fonts.
<img width="1195" alt="interface scaling 770 enabled 2" src="https://user-images.githubusercontent.com/31294363/147837681-53801b45-e6a2-4109-8bbc-52b97bd4a1ca.png">

**Comment by Bionus** [comment]
I was talking about other apps, such as Microsoft apps, not other Grabber versions.
How do font sizes compare? Trying on my W10 install, before Grabber was much smaller, while 7.7.0 has similar font sizes as other apps.

**Comment by uvhat** [comment]
7.7.0 looks off from any app and tool that i'm using. Here is comparison with msvc2013 (2019/vscode is same) with 100% scaling of code field and other and notepad++. Same with default notepad, powershell, explorer etc... 

Using 3840x2160 32" with 150% scaling in win10 settings. nightly 7.6.2 looks good and size is same with other apps.

<img width="1920" alt="interface scaling 770 enabled 3" src="https://user-images.githubusercontent.com/31294363/147839967-f8dea195-b5be-4006-a2ec-b6b37c507378.png">

If I set 100% scaling in windows settings both 7.6.2 nightly (and older builds) and 7.7.0 become similar.

<img width="1285" alt="interface scaling 770 enabled 3 100% windows scaling" src="https://user-images.githubusercontent.com/31294363/147840260-2448f77d-a0fb-4e7d-bc54-372864b8c01f.png">


Maybe better to make manual interface and fonts scaling without using Windows scaling? 
Current solution allows using too small batch download lines and presets and it hard to set and edit many line JS naming, and in any case is off from usual text or code editors interfaces with 150% (or any other) system scaling.

**Comment by uvhat** [comment]
Found some tweak - for correction DPI scaling need to create BAT file near Grabber.exe with:

```
set QT_SCALE_FACTOR = 0.7
Grabber. exe
```
and launch grabber from it. QT_SCALE_FACTOR correct DPI for anything, Fonts and Icons too.


But really wonder about custom DPI setting, I made example build (don't know how to extract saved settings or set QT_SCALE_FACTOR after working qapp instance created,  I used dummy instance before main). 

<img width="1324" alt="dpi scaling 2" src="https://user-images.githubusercontent.com/31294363/147866952-d2e5c5dd-5de9-4cde-b062-55d4f86a795e.png">
<img width="1122" alt="dpi scaling 1" src="https://user-images.githubusercontent.com/31294363/147866953-93481ab2-fa37-4095-931b-0c732e0c3aa8.png">

I tried just to change fonts scaling, but here is still issue with another elements (small icons for example), so just change scaling for fonts is not very good way, there must change dpi scaling for whole app.
